deliver/spec/upload_screenshots_spec.rb

```diff
@@ -0,0 +1,93 @@
+require 'deliver/upload_screenshots'
+require 'fakefs/spec_helpers'
+
+describe Deliver::UploadScreenshots do
+  describe "screenshot collection" do
```

Comment:
```suggestion
  describe "#collect_screenshots_for_languages (screenshot collection)" do
```
Is this correct ruby convention?